css/templates/lune/components/_header.css

```diff
@@ -0,0 +1,53 @@
+@import "../page-vars";
+
+.ampstart-headerbar {
+  justify-content: space-between
+
+  height: var(--layout-header-height-mobile);
+
+  /* Override ampstart default box shadow styles */
+  box-shadow: none;
+
+  /* header is fixed so should be moved onto a new GPU layer */
+  will-change: transform;
+}
+
+.ampstart-headerbar-home-link {
+  position: absolute;
```

Comment:
Using absolute positioning + transform to make sure that the title is always centered. Since the container has display: flex, the hamburger icon and the fixed elements on the right would make it impossible to center the title perfectly otherwise